shared/src/sources/balancer/pool_storage.rs

```diff
@@ -92,6 +61,13 @@ pub struct RegisteredStablePool {
     pub block_created: u64,
 }
 
+#[derive(Copy, Debug, Deserialize, Clone, Eq, PartialEq)]
+pub enum PoolType {
+    Stable,
+    Weighted,
+    Other,
+}
```

Comment:
It looks like the `Other` variant is never created so is there a point in having it?

Reply:
I agree this can be removed.